target_chains/solana/examples/send_usd/src/lib.rs

```diff
@@ -0,0 +1,71 @@
+use {
+    anchor_lang::{
+        prelude::*,
+        solana_program::{
+            native_token::LAMPORTS_PER_SOL,
+            system_instruction,
+        },
+    },
+    pyth_solana_receiver_sdk::price_update::{
+        get_feed_id_from_hex,
+        PriceUpdateV2,
+    },
+};
+
+declare_id!("2e5gZD3suxgJgkCg4pkoogxDKszy1SAwokz8mNeZUj4M");
+
+pub const MAXIMUM_AGE: u64 = 24 * 3600 * 365;
+pub const FEED_ID: &str = "0xef0d8b6fda2ceba41da15d4095d1da392a0d2f8ed0c6c7bc0f4cfac8c280b56d";
+
+#[program]
+pub mod my_first_pyth_app {
+    use {
+        super::*,
+        pyth_solana_receiver_sdk::price_update::VerificationLevel,
+    };
+
+    pub fn send(ctx: Context<Send>, amount_in_usd: u64) -> Result<()> {
+        let price_update = &mut ctx.accounts.price_update;
+        let price = price_update.get_price_no_older_than_with_custom_verification_level(
```

Comment:
Use `get_price_no_older_than` in this default example

Reply:
fixed